tutorials/commands/flux-job-cancel.rst

```diff
@@ -0,0 +1,114 @@
+.. _flux-job-cancel:
+.. _flux-job-cancelall:
+
+========================
+How to Cancel a Flux Job
+========================
+
+Inevitably submitted jobs will have to be canceled for one reason or another.  This tutorial
+will show you how.
+
+----------------------
+Basic Job Cancellation
+----------------------
+
+The basic way to cancel a job is through ``flux job cancel``.  All you have to do is specify
+the jobid on the command line.  Here is a simple example after submitting a job.
+
+.. code-block:: console
+
+    $ flux mini submit sleep 100
+    ƒh35Dh5qRyq
+
```

Comment:
Maybe between here we could add like:

```markdown
If you've job submit the job, the id will be readily available. If you need to see a list of your current jobs, you can use `flux jobs`.
```

Reply:
We also just merged `flux job last` which might be even easier than `flux jobs` in this case

Reply:
In this particular case I was mostly trying to show that the job was running.  Since `flux mini submit` already output the jobid and then afterwards it showed the job was canceled.